packages/xstate-test/src/types.ts

```diff
@@ -125,20 +130,137 @@ export interface TestEventConfig<TTestContext> {
    * ]
    * ```
    */
-  cases?: EventCase[];
+  cases?: Array<EventCase<TEvent>>;
+}
+
+export type TestEventsConfig<TState, TEvent extends EventObject> = {
+  [EventType in TEvent['type']]?:
+    | EventExecutor<TState, TEvent>
+    | TestEventConfig<TState, TEvent>;
+};
+
+export interface TestModelEventConfig<TState, TEvent extends EventObject> {
+  cases?:
+    | ((state: TState) => Array<EventCase<TEvent>>)
+    | Array<EventCase<TEvent>>;
+  exec?: EventExecutor<TState, TEvent>;
+}
```

Comment:
Minor quality of life thing I noticed while refactoring some experimental code to use new API:

When using `getPlanForEvents` it's not necessary to define `cases` since they are unsused. Previously, the events config allowed mapping to the `EventExecutor` directly without needing to wrap it in `{ exec: ... }` which was a nice convenience when you've got an object with tons of events. Often my event executors are one-liners, like the following:

```ts
'click': () => cy.findByRole('button', /click me/i).click(),
```

so it feels extra cumbersome to have to expand that to:

```ts
'click': {
  exec: () => cy.findByRole('button', /click me/i).click(),
},
```

especially when you've got dozens of these one-liners.

Reply:
I fixed this so that you can either specify an executor by itself, or in an `{ exec: ... }` object.